### DISCUSSION TITLE
Covers: open lit up and closed disabled if cover is open

### DISCUSSION THREAD

**Discussion by PyroghostX** [Clooos/Bubble-Card#294] [Feature requests]
Stock feature that i love and use all the time is if a cover is fully closed, then the closed is disabled and the open is active and vice versa, its very handy to not have to think when i want to close my curtains and blinds
only thing stopping me from using your cover card, it looks alot better than stock

![chrome_jbfHjJJ6i8](https://github.com/Clooos/Bubble-Card/assets/9093396/2292668b-9533-4cbc-bd7a-f4348e30dbf8)

**Comment by johndoe0815** [comment]
Just wanted to open this as a request, as I now started to try out bubble card. I agree, in HA standard tiles the respective cover buttons are deactivated, I am also missing this in bubble card.
Standard:
![grafik](https://github.com/Clooos/Bubble-Card/assets/92877294/479ceeae-4fd2-4b6b-8c3a-72377fded0d6)

Bubble Card:
![grafik](https://github.com/Clooos/Bubble-Card/assets/92877294/d687db34-f6c0-4f27-b8d5-37d63ebeca51)
